apps/financial-aid/web-veita/graphql/errorLink.ts

```diff
@@ -15,7 +15,7 @@ export default onError(({ graphQLErrors, networkError }: ErrorResponse) => {
     graphQLErrors.forEach((err) => {
       switch (err.extensions?.code) {
         case 'UNAUTHENTICATED':
-          return api.logOut('?villa=innskraning-utrunnin')
+          return //api.logOut('?villa=innskraning-utrunnin')
```

Comment:
This should be commented back.... riiiiight?

Reply:
Actually, noo i wanted to check whether this is the cause of the error in web-veitu login on dev.
It is also commented out in web-osk for some reason. 
I think while we are using fakeUsers we need this to be commented out.